docs/manuals/source/TasksAndConcepts/Plugins/StorageDaemonPlugins/ScsicryptoSd.rst.inc

```diff
@@ -83,29 +83,73 @@ Linux (SG_IO ioctl interface):
 
 The user running the storage daemon needs the following additional capabilities: :index:`\ <single: Platform; Linux; Privileges>`\
 
--  :strong:`CAP_SYS_RAWIO` (see capabilities(7))
+-  :strong:`CAP_SYS_RAWIO+EP` (see capabilities(7))
 
    -  On older kernels you might need :strong:`CAP_SYS_ADMIN`. Try :strong:`CAP_SYS_RAWIO` first and if that doesn’t work try :strong:`CAP_SYS_ADMIN`
 
 -  If you are running the storage daemon as another user than root (which has the :strong:`CAP_SYS_RAWIO` capability), you need to add it to the current set of capabilities.
 
--  If you are using systemd, you could add this additional capability to the CapabilityBoundingSet parameter.
+-  If you are using systemd, you could add this additional capability to the AmbientCapabilities=CAP_SYS_RAWIO parameter.
 
-   -  For systemd add the following to the bareos-sd.service: :strong:`Capabilities=cap_sys_rawio+ep`
+   -  For systemd add the following to the bareos-sd.service: :strong:`AmbientCapabilities=CAP_SYS_RAWIO`
+   -  To check status of capabilities of the running daemon you can use the :command:`getpcaps` followed by the pid of bareos-sd.
 
-You can also set up the extra capability on :command:`bscrypto` and :command:`bareos-sd` by running the following commands:
+.. warning::
+
+   As of systemd version 249 There's no mechanism to pass restricted flag (+ep), so the result will always be full CAP_SYS_RAWIO (eip)
+
+You can also set up the extra capability on :command:`bscrypto`,:command:`bareos-sd`,:command:`bls`,:command:`bextract` and :command:`btape` by running the following commands:
+
+.. code-block:: shell-session
+
+   setcap cap_sys_rawio=ep /usr/sbin/bscrypto
+   setcap cap_sys_rawio=ep /usr/sbin/bareos-sd
+   setcap cap_sys_rawio=ep /usr/sbin/bls
+   setcap cap_sys_rawio=ep /usr/sbin/bextract
+   setcap cap_sys_rawio=ep /usr/sbin/btape
```

Comment:
What about bcopy and bscan?
How about sorting the list of commands alphabetically?

Reply:
You make a very good point on those two, I wonder how I managed myself to forget them. +1 for the order. Will do that